docs/source/usage_guides/deepspeed.md

```diff
@@ -422,6 +422,13 @@ We will look at the changes needed in the code when using these.
 based on model, dataloaders, dummy optimizer and dummy schedulers provided to `prepare` method. 
 Only the `auto` fields specified in above examples are handled by `prepare` method and the rest have to be explicitly specified by the user.
```

Comment:
this is super-ambiguous - since there are multiple examples above and the last example has only one entry that can be used with `auto`

May I suggested to specify explicitly which variables are "eligible"?

Reply:
Make sense, will work on improving the docs of DeepSpeed.